README.md

```diff
@@ -583,6 +584,31 @@ export default function PostForm() {
 }
 ```
 
+NodeJS example:
+
+```js
+const client = new GraphQLClient({
+  url: 'https://domain.com/graphql',
+  cache: memCache(),
+  fetch: require('node-fetch'),
+  FormData: require('formdata-node')
+})
+
+const uploadPostPictureMutation = /* GraphQL */ `
```

Comment:
The editor hint `/* GraphQL */` is nice! For consistency could you add it to other queries in the docs or remove it from this one.